docs/en/install/install-dce.md

```diff
@@ -6,18 +6,21 @@ This page describes the installation procedure of DCE 5.0.
 
     Click to [watch installation video](../videos/install.md).
 
+    For offline installation, see [offline installation](offline-intall.md)
+
 ## Preparation
 
-- Prepare a k8s cluster. See [how to deploy a k8s cluster](install-k8s.md)
+- Prepare a k8s cluster.
 
     !!! note
 
-        - Available cluster resources: CPU > 10 cores, memory > 12 GB, remained storage space > 100 GB (currently, DCE 5.0 provides multiple replicas by default. If you choose a single replica, the resource consumption can be reduced to 2 cores and 4 GB)
-        - Kubernetes version: The latest stable version is recommended. It supports v1.21 to v1.24 currrently.
-        - Supported CRIs: Docker and containerd
-        - Storage: Prepare StorageClass and set it to the default. For details see [how to deploy a k8s cluster](install-k8s.md)
-        - Only support X86_64 currently
-        - Ensure your cluster has installed with CoreDNS
+        - Available cluster resources: CPU > 10 cores, memory > 12 GB, remained storage space > 100 GB (currently, DCE 5.0 provides multiple replicas by default. If you choose a single replica, the resource consumption can be reduced to 4 cores and 8 GB).
+        - Kubernetes version: The latest stable version is recommended. It supports v1.20 to v1.24 currrently.
+        - Supported CRIs: Docker and containerd.
+        - Storage: Prepare StorageClass and set it to the default.
+        - Only support X86_64 currently.
+        - Ensure your cluster has installed with CoreDNS.
+        - If a cluster with a node, ensure you have removed the taints from the node.
```

Comment:
"If a cluster with a node" 
A bit confusing, don't know what u mean if just read en version.

Reply:
Has changed to:

...removed the taint labels from ...